node/src/manager/commands/config.rs

```diff
@@ -177,6 +177,7 @@ pub async fn provider(
         registry,
         metrics,
         Arc::new(NoopIdentValidator),
+        false,
```

Comment:
Don't you want it dependant on the config in graphman too?

Reply:
only for run, the others don't depend on the ProviderManager component